AzureCalling/Controllers/LobbyViewController.swift

```diff
@@ -225,12 +224,16 @@ class LobbyViewController: UIViewController, UITextFieldDelegate {
             fatalError("Unexpected destination: \(destination)")
         }
 
+        let isGroupIdValid = !(groupId?.isEmpty ?? true)
+        let validGroupId = isGroupIdValid ? groupId! : UUID().uuidString
+
         cleanRenderView()
-        let joinCallConfig = JoinCallConfig(joinId: joinInput,
-                                            isMicrophoneMuted: toggleMicrophoneButton.isSelected,
-                                            isCameraOn: !toggleVideoButton.isSelected,
-                                            displayName: displayName,
-                                            callType: joinCallType)
```

Comment:
We can continue to use `callType`, but for now there will only be one type. This will make it easier to merge later on.